mlflow/langchain/__init__.py

```diff
@@ -329,6 +346,28 @@ def log_model(
             type(lc_model.agent.llm_chain.llm).__name__,
         )
 
+    if isinstance(lc_model, langchain.chains.RetrievalQA):
+        if conda_env is None:
+            logger.warning(
+                "RetrievalQA models require a conda environment. "
```

Comment:
Does `RetrievalQA` only work in a conda environment?

Reply:
Actually no. pip should also work.